src/coreclr/interpreter/interpconfigvalues.h

```diff
@@ -0,0 +1,15 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#ifdef DEBUG
+#define CONFIG_STRING(name, key)                RELEASE_CONFIG_STRING(name, key)
+#else
+#define CONFIG_STRING(name, key)
+#endif
+
+RELEASE_CONFIG_STRING(Interpreter, "Interpreter")
+CONFIG_STRING(InterpHalt, "InterpHalt");
```

Comment:
A nit - JIT has the same control knob and names it JitBreakEmit. It would be nice to use the same kind of naming (InterpBreakEmit).

Reply:
Hmm, my attempt was using the same name. Using `DOTNET_JitHalt` emitted a special break asm instruction at the beginning of the method, breaking into the debugger at runtime, when the method is executed. `JitBreakEmit` seems to do something else, breaking during compilation / code generation.

Reply:
I am sorry, you are right. I've looked at the wrong settings, the clrconfigvalues.h and there was just the JitBreakEmit, so I've mistakenly thought it is the right name.